README.md

```diff
@@ -142,7 +148,7 @@ This is a type for representing string values.
 The `get` method returns the value of the `StringType` instance.
 
 ```javascript
-var stringType = new minim.StringType('foobar');
+var stringType = new minim.StringType({}, {}, 'foobar');
```

Comment:
One thing we could do is do this backwards:

```javascript
var stringType = new minim.StringType('foobar', {}, {});
```

Where the first argument was the content of the element.